InomialClient.js

```diff
@@ -194,6 +260,78 @@ class InomialClient {
         this.sendRequest(request);
     }
 
+    logWireRequest(graphqlRequest) {
```

Comment:
Sorry but I just want you to get rid of all of this. If we're in verbose mode then print the request, but don't *process* it as well. Just log the basics. Keep it simple.

There are 71 lines of code here which could all be buggy. It includes complex regular expressions. It is not the purpose of the client to do this.